vespaclient-java/src/test/java/com/yahoo/vespaget/CommandLineOptionsTest.java

```diff
@@ -80,7 +79,6 @@ public void testValidOptions() {
                 "--noretry",
                 "--trace", "1",
                 "--priority", Integer.toString(DocumentProtocol.Priority.HIGH_3.getValue()),
-                "--loadtype", "dummyloadtype",
```

Comment:
Used in any system tests?

Reply:
I see there is a lone load types test under `tests/vds`. Dropkicking it in https://github.com/vespa-engine/system-test/pull/2336. I'll wait to merge this PR until the test is removed to avoid any unnecessary breakage.